Presentation.Web/Tests/01_Parallel/It-Contract/LocalAdmin.ItContract.UiCustomization.e2e.spec.ts

```diff
@@ -61,14 +62,16 @@ describe("Local admin is able customize the IT-Contract UI", () => {
             .then(() => testFieldCustomization(contractName, "ItContracts.frontPage.isActive", ContractNavigationSrefs.frontPageSref, "contractIsActive"))
 
             // Deadlines tab
-            .then(() => testFieldCustomization(contractName, "ItContracts.deadlines.agreementDeadlines", ContractNavigationSrefs.deadlinesPageSref, "agreement-deadlines"))
-            .then(() => testFieldCustomization(contractName, "ItContracts.deadlines.termination", ContractNavigationSrefs.deadlinesPageSref, "termination"))
+            .then(() => testFieldCustomizationWithSubtreeIsComplete(contractName, "ItContracts.deadlines.agreementDeadlines", ContractNavigationSrefs.deadlinesPageSref, "agreement-deadlines"))
+            .then(() => testFieldCustomizationWithSubtreeIsComplete(contractName, "ItContracts.deadlines.termination", ContractNavigationSrefs.deadlinesPageSref, "termination"))
+            .then(() => testTabCustomizationWithSubtreeIsComplete(contractName, ["ItContracts.deadlines.agreementDeadlines", "ItContracts.deadlines.termination"], ContractNavigationSrefs.deadlinesPageSref))
```

Comment:
what about the regular "tab" on/off tests? they are still relevant - not only that the sub tree can disable the parent

Reply:
the regular "tab" tests are still there, they are present above the "field" tests
![image](https://user-images.githubusercontent.com/101557040/189314394-4f774f7d-e78f-47e9-8da1-a27651ab403d.png)